config/application.yml.default

```diff
@@ -9,7 +9,7 @@ aws_http_timeout: '5'
 http_read_timeout: '5'
 http_open_timeout: '5'
 
-ca_issuer_host_allow_list: 'aia.certipath.com,repo.fpki.gov,crl.disa.mil,http.fpki.gov,ssp-aia.symauth.com,pki.treasury.gov,rootweb.managed.entrust.com,aia1.ssp-strong-id.net,pki.treas.gov,crls.pki.state.gov,sspweb.managed.entrust.com,nfirootweb.managed.entrust.com,ssp-aia.digicert.com'
+ca_issuer_host_allow_list: 'aia.certipath.com,repo.fpki.gov,crl.disa.mil,http.fpki.gov,ssp-aia.symauth.com,pki.treasury.gov,rootweb.managed.entrust.com,aia1.ssp-strong-id.net,pki.treas.gov,crls.pki.state.gov,sspweb.managed.entrust.com,nfirootweb.managed.entrust.com,ssp-aia.digicert.com,www.fis.evincible.com'
```

Comment:
do we need to update the domain allowlist for this?

Reply:
The production/deployed environment list is below, and I think we override it in `production` via S3 too so it's not strictly necessary.

Reply:
ok sweet, I always forget which configs in this repo get blocked by the firewall and which don't

Reply:
Hi, I may be wrong but on an Exostar certificate the domain is marked as fis.evincible.com not www.fis.evincible.com, they may be interchangeable but I just wanted to make sure.

Reply:
This is the list for development and stuff where there isn't a firewall and has tests run against it to make sure we aren't missing any hosts by default (and as a reference to make updates to the deployed environment allowlist if needed)